operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1194,6 +1194,17 @@ spec:
               strimzi.io/kind: cluster-operator
           template:
             metadata:
+              annotations:
```

Comment:
Just to note, these could be references to the internal images and non-SHA tags. OSBS will handle the replacements, which might simplify the bundle automation script.

Reply:
For the initial bundle releases, like for 2.8.0-0, wouldn't we need the SHAs here to ensure the correct images are used?

Reply:
The configuration in container.yaml tells OSBS to convert the internal repositories and the image tags to the external repositories + SHA.

```
operator_manifests:
  enable_digest_pinning: true
  enable_repo_replacements: true
  enable_registry_replacements: true
  repo_replacements:
    - registry: registry-proxy.engineering.redhat.com
      package_mappings:
        amqstreams-operator-container: amq-streams/strimzi-rhel9-operator
        amqstreams-kafka-36-container: amq-streams/kafka-36-rhel9
        amqstreams-kafka-37-container: amq-streams/kafka-37-rhel9
        amqstreams-bridge-container: amq-streams/bridge-rhel9
        amqstreams-drain-cleaner-container: amq-streams/drain-cleaner-rhel9
        amqstreams-operator-bundle-container: amq-streams/strimzi-operator-bundle
```

Reply:
+1 to using tags.

Reply:
> The configuration in container.yaml tells OSBS to convert the internal repositories and the image tags to the external repositories + SHA.

When we need to rebuild/release the bundle (e.g. for bundle metadata/cosmetic changes) without updating the image reference, how do we ensure the OSBS uses the same images if we use tags?

Using the image tags that include build number suffix? e.g. 2.7.0-4

Reply:
Wait..  there's a part I don't I understand.  In the current release process what it making this commit?  I think it'll need to be updated too to reflect the fact that the pull specs have moved about/

https://github.com/jboss-container-images/amqstreams-1-openshift-image/commit/662348e1099567be5350562e736b0803a42a1192

Reply:
That commit is made by the bundle automation [1] which is executed by CPaaS `render_templates` script.  And you are right, I missed some changes that need to happen there for the scripts to find the shas to replace now that `relatedImages` section is removed

Let me update that

[1] https://github.com/jboss-container-images/amqstreams-1-openshift-image/tree/amqstreams28-dev/operator-metadata/automation

Reply:
> Using the image tags that include build number suffix? e.g. 2.7.0-4

I think yes. The build_info environment variables that the automation scripts are currently using (See `create_sha_dict`) should have the full image reference that includes the tag with the build number.

Reply:
Alright, that should work, latest commit shows what it would look like if we used tags instead f digests. Updated the build automation to account for these changes as well